Pipfile

```diff
@@ -17,14 +17,14 @@ bleach = ">=3.0.2"
 python-dateutil = "*"
 unidecode = "*"
 celery = "==4.1.0"
-arxiv-auth = "==0.4.1"
+arxiv-auth = ">=0.4.1"
```

Comment:
I'm not sure we ever settled on a practice with regards to pinning, but it might be better to pin to the exact versions for both `arxiv-auth` and `arxiv-base`.